packages/retail-ui/components/RenderContainer/RenderContainer.tsx

```diff
@@ -1,89 +1,89 @@
 import * as React from 'react';
 import ReactDOM from 'react-dom';
 
-let lastID = 0;
-function nextID() {
-  return ++lastID;
-}
-function setID(id: number) {
-  lastID = id;
-}
-
 const REACT_16 = !!ReactDOM.createPortal;
 
 export default class RenderContainer extends React.Component<{
   anchor?: React.ReactNode;
 }> {
-  private _domContainer: HTMLElement;
+  public static lastId = 0;
 
-  private _testID: number;
+  private domContainer: HTMLElement;
+  private rootId: number;
 
   constructor(props: {}, context: {}) {
```

Comment:
```suggestion
  constructor(props: { anchor?: React.ReactNode }) {
```

или лучше объявить интерфейс пропсов